data/level_to_cpm.json

```diff
@@ -0,0 +1,81 @@
+{
+  "1":    0.094,
```

Comment:
@earthchie cp_multiplier and additional_cp_multiplier are used when available (for pokemons).
this data is used to get CPM at specific level or get level of pokemon by it's CPM